### ISSUE TITLE
NPE inside `ASTNodeUtils.findLBrace`

### ISSUE THREAD

**Issue by petertrr** [cqfn/diKTat#137]
```
[INFO]    [ktlint] C:\Users\pwx5333130\Documents\diktat-fresh\diktat-common\src\main\kotlin\org\cqfn\diktat\common\config\rules\RulesConfigReader.kt:18:1: Internal Error (diktat-ruleset:empty-block-structure). Please create a ticket at https://github.com/pinterest/ktlint/issues (if possible, provide the source code that triggered an error) (cannot be auto-corrected)
[INFO]    [ktlint] C:\Users\pwx5333130\Documents\diktat-fresh\diktat-common\src\test\kotlin\org\cqfn\diktat\test\JsonReaderTest.kt:11:40: Internal Error (diktat-ruleset:empty-block-structure). Please create a ticket at https://github.com/pinterest/ktlint/issues (if possible, provide the source code that triggered an error) (cannot be auto-corrected)
```

The problem code (as by offset provided from the error):
```
@JsonIgnoreProperties(ignoreUnknown = true)
```
```
filter { it.name == "CLASS_NAME_INCORRECT" && it.enabled }
```

Here is the error:
```
[INFO]    [ktlint] Caused by: kotlin.KotlinNullPointerException
[INFO]    [ktlint] 	at org.cqfn.diktat.ruleset.utils.AstNodeUtilsKt.findLBrace(AstNodeUtils.kt:324)
[INFO]    [ktlint] 	at org.cqfn.diktat.ruleset.rules.EmptyBlock.searchNode(EmptyBlock.kt:42)
[INFO]    [ktlint] 	at org.cqfn.diktat.ruleset.rules.EmptyBlock.visit(EmptyBlock.kt:38)
[INFO]    [ktlint] 	at com.pinterest.ktlint.core.KtLint$lint$1.invoke(KtLint.kt:167)
[INFO]    [ktlint] 	... 18 more
```

**Comment by petertrr** [comment]
Closed in #198

**Comment by petertrr** [comment]
Still happens, e.g. https://github.com/cqfn/diKTat/runs/1032675230
```
[INFO]    [ktlint] /home/runner/work/diKTat/diKTat/diktat-common/src/main/kotlin/org/cqfn/diktat/common/config/rules/RulesConfigReader.kt:18:1: Internal Error (diktat-ruleset:empty-block-structure). Please create a ticket at https://github.com/pinterest/ktlint/issues (if possible, provide the source code that triggered an error) (cannot be auto-corrected)
```
